docs/release-notes/2.39.0.md

```diff
@@ -0,0 +1,65 @@
+
+![clouds](../_images/PMM_QAN_metadata.png)
+
+
+## Improvements
+
+- [PMM-6279](https://jira.percona.com/browse/PMM-6279) - [QAN]: Query metadata not visible in new UI
+- [PMM-10270](https://jira.percona.com/browse/PMM-10270) -[Backup Management]: Enhanced MongoDB Backup Management to support restoring to different replica set clusters with the same configuration.
+- [PMM-11999](https://jira.percona.com/browse/PMM-11999) - [Advisors]: Expose check family field in advisors API
+- [PMM-12257](https://jira.percona.com/browse/PMM-12257) - [Advisors]: The **Advisors** page now includes a new **Technology** column, showing the database type for each advisor in the table. This helps you easily differentiate between the MongoDB, MySQL and PostgreSQL	advisors.
+- [PMM-11637](https://jira.percona.com/browse/PMM-11637) - In PMM 2.39.0, we updated the interface to reflect the new PMM logo as per the new branding.
+- [PMM-11206](https://jira.percona.com/browse/PMM-11206) - [Alerting]: Enhanced the **Add rule template** dialog box to support YAML content for multiple alert templates. Alert templates added in bulk are displayed individually on the **Alert rule templates** page.
+ 
+## Components updates
+- [PMM-12070](https://jira.percona.com/browse/PMM-12070) - We have updated the deprecated [Boomtable plugin](https://github.com/percona/grafana-dashboards/tree/main/panels/yesoreyeram-boomtable-panel) to the Grafana table plugin.
+ 
+## Bugs Fixed
+
+- [PMM-12151](https://jira.percona.com/browse/PMM-12151) - [QAN]: Explain plan not displayed if database name is missing in the query
+- [PMM-10391](https://jira.percona.com/browse/PMM-10391) - PMM did not work with external clickhouse versions greater than 22.5.2. The issue has been resolved now.
+- [PMM-11658](https://jira.percona.com/browse/PMM-11658) - pmm-agent crashes simultaneously on all nodes
+- [PMM-11992](https://jira.percona.com/browse/PMM-11992) - Database name is ignored while adding any Postgres
+- [PMM-12161](https://jira.percona.com/browse/PMM-12161) - "MongoDB Cluster Summary" - too many "Config Services"
+- [PMM-12231](https://jira.percona.com/browse/PMM-12231) - PMM disallows to install, update and remove plugins
+- [PMM-12263](https://jira.percona.com/browse/PMM-12263) - Wrong Replication Lag by Set values if RS is PSA - MongoDB Cluster Summary
+- [PMM-12280](https://jira.percona.com/browse/PMM-12280) - Fixed an issue where the **Service Summary Panel** on Home Dashboard used the incorrect units to display `Available Memory`.
+- [PMM-10665](https://jira.percona.com/browse/PMM-10665) -  Fixed issue caused QAN to show incorrect values when setting non-default values for performance schema. Make sure to restart pmm-agent after changing the settings for perfschema in MySQL.
```

Comment:
Do we have a note about restart in the documentation (not only in RN)
Can be one of the top bugs, very important